exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcMetricExporter.java

```diff
@@ -282,24 +292,43 @@ private Builder() {}
     protected Builder fromConfigMap(
         Map<String, String> configMap, NamingConvention namingConvention) {
       configMap = namingConvention.normalize(configMap);
-      Long value = getLongProperty(KEY_METRIC_TIMEOUT, configMap);
+
+      Long value = getLongProperty(KEY_TIMEOUT, configMap);
+      if (value == null) {
+        value = getLongProperty(KEY_TIMEOUT.replace("metric", ""), configMap);
```

Comment:
Rather than the clever replacing, I think I'd personally prefer just having another constant defined with the non-metric value. To me, that makes the code a little easier to read. Also, the common properties could be put somewhere common and shared between the metric/span exporter config.

Reply:
Done in 34c8c25